recipes/pypyodbc/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pypyodbc" %}
+{% set version = "1.3.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.2.zip"
+  sha256: 90a9a1fdbe5c06b875155d82562beb50bb22743190ca9b46a2cb71030ee65496
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - pypyodbc
+  requires:
+    - unixodbc  # [unix]
```

Comment:
Just importing this requires `unixodbc`?

Reply:
Yes: https://github.com/jiangwen365/pypyodbc/blob/ae109cba8602a7bfd87a4237acab30cef8df1225/pypyodbc.py#L425-L448

Theoretically, the program should be able to work with multiple odbc drivers (even proprietary ones). Should we force unixodbc on the end users?

Reply:
I'm not sure. Somebody else will have to chime in

Reply:
Hmm... the comments in the source code mention only UnixODBC. I'll simply add it as a dep, similar to what I did for [pyodbc](https://github.com/conda-forge/pyodbc-feedstock/blob/master/recipe/meta.yaml#L29).